dask_cuda/tests/test_device_host_file.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import cupy
+from dask_cuda.device_host_file import DeviceHostFile
+from random import randint
+
+import pytest
+from dask_cuda.utils_test import gen_random_key
+from cupy.testing import assert_array_equal
+
+
+@pytest.mark.parametrize("num_host_arrays", [1, 10, 100])
+@pytest.mark.parametrize("num_device_arrays", [1, 10, 100])
+@pytest.mark.parametrize("array_size_range", [(1, 1000), (100, 100), (1000, 1000)])
+def test_device_host_file_short(num_device_arrays, num_host_arrays, array_size_range):
+    import tempfile
+
+    tmpdir = tempfile.TemporaryDirectory()
```

Comment:
You may also want to learn about the `tmp_path` fixture that comes with pytest

https://docs.pytest.org/en/latest/tmpdir.html

Reply:
Thanks, I didn't know about this. Done too.